app/views/helpers/signUpCompleteIndividualCurrentTaxYearHelper.scala.html

```diff
@@ -0,0 +1,36 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.AppConfig
+
+@(endYearOfCurrentTaxPeriod: Int, updatesBefore: List[(String, String)], updatesAfter: List[(String, String)])(implicit messages: Messages, appConfig: AppConfig)
+
+
+@if(updatesBefore.nonEmpty){
+    <li>@Html(Messages("sign-up-complete.whatHappensNow.sendPreviousUpdates"))</li>
+    <ul class="list list-bullet">@for((updatesBefore, year) <- updatesBefore) {
```

Comment:
```suggestion
    <ul class="list list-bullet">@for((update, year) <- updatesBefore) {
```
Could this be updated so the name of the value in the list is not the same as the name of the list?